Events/Christmas 201X.lua

```diff
@@ -510,6 +509,11 @@ monster.frame = function(self, id, isAttack)
 end
 
 monster.moveAround = function(self, movement, dontMoveWith, radius)
+	--[[Moves the monster around.
+	'movement' is a value from movementTypes,
+	'dontMoveWith' and 'radius' are the needed players in the given radius to make the monster not move.
+	dontMoveWith=1 and radius=50 will check if there is at least 1 player in a radius of 50px, if so, there will not be any movement.
+	]]
```

Comment:
It's not necessary to documentate event functions, unless it was a repository

How about renaming `dontMoveWith` to `minimumMice`?

Reply:
It is still better to have a documentation for some functions if they're not as intuitive as you say. I guess that "dontMoveWith" is better than "minimumMice" tho

Reply:
dontMoveWith gives the idea of callback